src/io/DisposableSeekableHandle.php

```diff
@@ -0,0 +1,17 @@
+<?hh
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\Experimental\IO;
+
+use namespace HH\Lib\Experimental\Fileystem;
```

Comment:
unused (ditto in several other files) - also, the typo will help for finding the unused use statements :)